### PR TITLE
[ValloxMV] Adding Co2 measurement

### PR DESCRIPTION
**Pull request by bjoernbrings**
For ValloxMV binding I added a channel for CO2 measurement and corrected one bug for humidity measurement (in fact it was working but seems like it was using a wrong register).

I don't have a unit with CO2 measurement but it has been tested by a user [here](https://community.openhab.org/t/vallox-mv-binding-co2-reading/118782/6).

Signed-off-by: bjoernbrings <bjoernbrings@web.de>

### CODE REVIEW

File: bundles/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVWebSocket.java [hunk 1]
```diff
@@ -503,6 +505,7 @@ public void onBinary(InputStream in) {
                 updateChannel(ValloxMVBindingConstants.CHANNEL_TEMPERATURE_INCOMING,
                         new QuantityType<>(bdTempIncoming, SIUnits.CELSIUS));
                 updateChannel(ValloxMVBindingConstants.CHANNEL_HUMIDITY, new QuantityType<>(iHumidity, Units.PERCENT));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_CO2, new DecimalType(iCo2));
```
Comment (fwolter) [inline bundles/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVWebSocket.java:508]:
```suggestion
                updateChannel(ValloxMVBindingConstants.CHANNEL_CO2, new QuantityType<>(iHumidity, Units.PARTS_PER_MILLION));
```

File: bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/i18n/valloxmv_de.properties [hunk 2]
```diff
@@ -37,6 +37,8 @@ channel-type.valloxmv.tempincoming.label = Temperatur Zuluft
 channel-type.valloxmv.tempincoming.description = Aktuelle Temperatur der Zuluft
 channel-type.valloxmv.humidity.label = Luftfeuchtigkeit
 channel-type.valloxmv.humidity.description = Aktuelle Luftfeuchtigkeit der Abluft
+channel-type.valloxmv.co2.label = Co2
```
Comment (fwolter) [inline bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/i18n/valloxmv_de.properties:40]:
```suggestion
channel-type.valloxmv.co2.label = CO2
```

File: bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/i18n/valloxmv_de.properties [hunk 3]
```diff
@@ -37,6 +37,8 @@ channel-type.valloxmv.tempincoming.label = Temperatur Zuluft
...
+channel-type.valloxmv.co2.description = Aktueller Kohlenstoffdioxid Gehalt in der Abluft
```
Comment (fwolter) [inline bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/i18n/valloxmv_de.properties:41]:
```suggestion
channel-type.valloxmv.co2.description = Aktueller Kohlenstoffdioxidgehalt in der Abluft
```

File: bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/thing/thing-types.xml [hunk 4]
```diff
@@ -18,6 +18,7 @@
 			<channel id="tempincomingbeforeheating" typeId="tempincomingbeforeheating"/>
 			<channel id="tempincoming" typeId="tempincoming"/>
 			<channel id="humidity" typeId="humidity"/>
+            <channel id="co2" typeId="co2"/>
```
Comment (fwolter) [inline bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/thing/thing-types.xml:21]: The rest of the file uses tabs for indentation, although spaces are correct. You could invoke `mvn spotless:apply` to fix the formatting.

File: bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/thing/thing-types.xml [hunk 5]
```diff
@@ -151,6 +152,13 @@
 		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
 	</channel-type>
 
+    <channel-type id="co2">
+        <item-type>Number:Dimensionless</item-type>
+        <label>CO2</label>
+        <description>CO2 measurement</description>
+        <state readOnly="true" pattern="%d"/>
```
Comment (fwolter) [inline bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/thing/thing-types.xml:159]:
```suggestion
        <state readOnly="true" pattern="%d %unit%"/>
```

**Review by fwolter — CHANGES_REQUESTED** [review]

**Comment by bjoernbrings** [comment]
Thanks for your input

**Review by fwolter — APPROVED** [review]
LGTM

**Pull request merged**